.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Expected behavior
```

Comment:
I suggest adding an additional bullet that asks them to describe their environment and version numbers (e.g. React, Angular, Vue, etc). As we know this repo will be used by a bunch of different frameworks.

Reply:
I'm down with that.  Coming soon.